src/Apps/ArtAdvisor/07-Curated-Discovery/App.tsx

```diff
@@ -0,0 +1,132 @@
+import { Box, Button, Join, Spacer, Text } from "@artsy/palette"
+import { FC, useReducer } from "react"
+import { Artists } from "./Components/Artists"
+import { Interests } from "./Components/Interests"
+import { Goals } from "./Components/Goals"
+
+export type State = {
+  goals: string[]
+  goalsFreeText: string
+  interests: string[]
+  interestsFreeText: string
+  artists: string[]
+  artistsFreeText: string
+}
```

Comment:
Keeping the modeling simple, just a bunch of `string[]` arrays corresponding to the suggestion pills, and some `string`s corresponding to the freetext fields.

Upon form submission, we'd shape up this payload into something suitable for an LLM call.

(Posted a follow-up question [here](https://artsy.slack.com/archives/C06SSV1K10D/p1718722761894599))